src/duo-auth.md

```diff
@@ -110,57 +110,46 @@ The application-specific configuration information retrieved from Duo
 must be added to `guacamole.properties` to describe how Guacamole
 should connect to the Duo service:
 
-duo-api-hostname
+`duo-api-hostname`
 : The hostname of the Duo API endpoint to be used to verify user identities.
-  This will usually be in the form `api-XXXXXXXX.duosecurity.com`, where
-  "XXXXXXXX" is some arbitrary alphanumeric value assigned by Duo. This
-  value will have been generated by Duo when you added Guacamole as an "Web
-  SDK" application, and can be found within the application details in the
-  "API hostname" field. *This value is required.*
+  This will usually be in the form {samp}`api-{XXXXXXXX}.duosecurity.com`,
+  where {samp}`{XXXXXXXX}` is some arbitrary alphanumeric value assigned by
+  Duo. This value will have been generated by Duo when you added Guacamole as
+  an "Web SDK" application, and can be found within the application details in
+  the "API hostname" field. *This value is required.*
 
-duo-integration-key
+`duo-client-id`
 : The integration key provided for Guacamole by Duo. This value will
   have been generated by Duo when you added Guacamole as an "Web SDK"
   application, and can be found within the application details in the
-  "Integration key" field. *This value is required and must be EXACTLY
-  20 characters.*
+  "Client ID" field. *This value is required.*
```

Comment:
* Is this still an "integration key", or should this first line be changed to something like `The client identifier provided for Guacamole...`?
* While we're here modifying this block, perhaps change "an" to "a": `a "Web SDK"`

Reply:
Oops - will fix both of these.